README.md

```diff
@@ -911,16 +917,18 @@ It is also necessary to fill out the `google` stanza in the configuration file.
 
 ```hocon
 google {
-  authScheme = "service"
+  applicationName = "cromwell"
+  
+  cromwellAuthenticationScheme = "service_account"
 
-  // If authScheme is "user"
+  // If cromwellAuthenticationScheme is "user_account"
```

Comment:
I prefer this the way you have it to make clear there are two distinct types of authentication.